test/interchain.go

```diff
@@ -156,33 +155,32 @@ type InterchainLink struct {
 // AddLink adds the given link to the Interchain.
 // If any validation fails, AddLink panics.
 func (ic *Interchain) AddLink(link InterchainLink) *Interchain {
-	if _, exists := ic.chains[link.Chain1]; !exists {
-		cfg := link.Chain1.Config()
-		panic(fmt.Errorf("chain with name=%s and id=%s was never added to Interchain", cfg.Name, cfg.ChainID))
-	}
-	if _, exists := ic.chains[link.Chain2]; !exists {
-		cfg := link.Chain2.Config()
-		panic(fmt.Errorf("chain with name=%s and id=%s was never added to Interchain", cfg.Name, cfg.ChainID))
-	}
+	// if _, exists := ic.chains[link.Chains[0].(ibc.Chain)]; !exists {
+	// 	cfg := link.Chains[0].(ibc.Chain).Config()
+	// 	panic(fmt.Errorf("chain with name=%s and id=%s was never added to Interchain", cfg.Name, cfg.ChainID))
+	// }
+	// if _, exists := ic.chains[link.Chains[1].(ibc.Chain)]; !exists {
+	// 	cfg := link.Chains[1].(ibc.Chain).Config()
+	// 	panic(fmt.Errorf("chain with name=%s and id=%s was never added to Interchain", cfg.Name, cfg.ChainID))
+	// }
```

Comment:
lets remove these comments if not necessary.

Reply:
done.